Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1164,7 +1168,7 @@ void TextureCache::TCacheEntryBase::SetEfbCopy(u32 stride)
 	is_efb_copy = true;
 	memory_stride = stride;
 
-	_assert_msg_(VIDEO, memory_stride >= CacheLinesPerRow(), "Memory stride is too small");
+	_assert_msg_(VIDEO, memory_stride >= BytesPerRow(), "Memory stride is too small");
```

Comment:
Oh yeah, this is a bug. I fixed it in a commit which never got pushed to master.

memory_stride >= BytesPerRow() is correct.